js_modules/dagster-ui/packages/ui-core/src/asset-graph/useAssetGraphData.tsx

```diff
@@ -203,7 +203,7 @@ export function useAssetGraphData(opsQuery: string, options: AssetGraphFetchScop
         supplementaryData,
       },
       spawnComputeGraphDataWorker,
-      options.useWorker ?? true,
```

Comment:
This doesn't seem related to the pyright change and it seems useWorker should be removed if it's falsy. maybe this was for testing?